packages/components/src/components/contract-card/contract-card-items/contract-card-item.jsx

```diff
@@ -2,10 +2,16 @@ import classNames from 'classnames';
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const ContractCardItem = ({ className, children, header, is_crypto, is_loss, is_won }) => {
+const ContractCardItem = ({ className, children, header, is_crypto, is_loss, is_won, is_long }) => {
```

Comment:
I was wondering do we still need `is_long` ? This property is not passed to ContractCardItem + it's not in `ContractCardItem.propTypes` (line 28). Please correct me if I'm mistaken